vector/src/main/java/im/vector/util/SlashCommandsParser.java

```diff
@@ -44,39 +44,63 @@
 
     private static final String LOG_TAG = SlashCommandsParser.class.getSimpleName();
 
-    // defines the command line operations
-    // the user can write theses messages to perform some room events
-    public static final String CMD_EMOTE = "/me";
-
-    // <user-id> [reason]
-    private static final String CMD_BAN_USER = "/ban";
-
-    // <user-id>'
-    private static final String CMD_UNBAN_USER = "/unban";
-
-    // <user-id> [<power-level>]
-    private static final String CMD_SET_USER_POWER_LEVEL = "/op";
-
-    // <user-id>
-    private static final String CMD_RESET_USER_POWER_LEVEL = "/deop";
+    public enum SlashCommand {
+
+        // defines the command line operations
+        // the user can write theses messages to perform some room events
+        EMOTE("/me", "<message>", VectorApp.getInstance().getString(R.string.command_description_emote)),
```

Comment:
You should let the enum have a ResInt instead String :
EMOTE("/me", "<message>", R.string.command_description_emote)